### PR TITLE
Add Transposition Cryptography Algorithms: Rail Fence, Simple Columnar, and Simple Columnar with Multiple Rounds 

### PR DESCRIPTION
**Pull request by Sejal-G**
### Have you read the [Contributing Guidelines on Pull Requests]
YES

### Description
Implemented the following Transposition Cryptography Algorithms in Cpp: 
1. Rail Fence Cipher Technique
2. Simple Columnar Transposition Technique, and 
3. Simple Columnar Transposition Multiple Rounds. 
The algorithms are commented to help beginners understand the problem concisely. 

### Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've implemented Transposition Cryptography Algorithms in the Cryptography section under C-Plus-Plus and updated the readme.md file.  

## Related Issues or Pull Requests
#1494

### CODE REVIEW

File: C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+int main() {
+
+	cout<<"Enter plain text"<<endl;
+	string s;
+	cin>>s;
+	string enc = encrypt(s);
+	cout<<"Cipher text: "<<enc<<endl;
+
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp]: Add a line at the end of the file.

File: C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		s2+=s[i+1];
+	}
+	return s1+s2;
+}
+
+int main() {
+
+	cout<<"Enter plain text"<<endl;
+	string s;
+	cin>>s;
```
Comment (HarshCasper) [inline C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp]: Format the Code properly. Put apt spaces around the operators.

File: C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	for(int i=0;i<5;i++){
+		cin>>order[i];
+	}
+	//MULTIPLE ROUNDS
+	for(int i=0;i<rounds;i++){
+		s = encrypt(s,order);
+	}
+	cout<<s;
+	
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp]: Add a new line at the end of file

File: C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+	
+	cout<<"Enter the order of repositioning columns: ";  //Example(1 0 2 3 4)
+	for(int i=0;i<5;i++){
+		cin>>order[i];
+	}
+	//MULTIPLE ROUNDS
+	for(int i=0;i<rounds;i++){
+		s = encrypt(s,order);
+	}
+	cout<<s;
```
Comment (HarshCasper) [inline C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp]: Have consistent indentation and formatting

File: C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+			encString+=arr[j][order[i]];
+		}	
+	}
+	return encString;
+}
+
+int main(){
+	
+    string s;int rounds;vector<int>order(5); 
+	cout<<"Enter the plain text: "; //(suggested with length greater than 5)
```
Comment (HarshCasper) [inline C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp]: Use comments above the line of code not beside them

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp (hunk 4 above)
Comment (Sejal-G) [reply C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp]: Done. Thank you for suggesting the changes.

File: C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp (hunk 5 above)
Comment (Sejal-G) [reply C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp]: Done. Thank you for suggesting the changes.

File: C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp (hunk 2 above)
Comment (Sejal-G) [reply C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp]: Done. Thank you for suggesting the changes.

File: C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp (hunk 1 above)
Comment (Sejal-G) [reply C-Plus-Plus/cryptography/Rail_fence_Cipher.cpp]: Done. Thank you for suggesting this change.

File: C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp (hunk 3 above)
Comment (Sejal-G) [reply C-Plus-Plus/cryptography/Simple_Columnar_Transposition_multiple_rounds.cpp]: Done. Thank you for suggesting this change.

**Comment by Sejal-G** [comment]
Thank you for suggesting the changes. Kindly review the updated commit.

**Comment by HarshCasper** [comment]
Few changes: 

- Add a Line at the end of the Code. 
- Make apt README changes. 
- Add beginner pitch to the Code so that we know about the Algorithm.

**Comment by Sejal-G** [comment]
> Few changes:
> 
> * Add a Line at the end of the Code.
> * Make apt README changes.
> * Add beginner pitch to the Code so that we know about the Algorithm.

Done. Thank you for suggesting the changes. Kindly review the code.

**Comment by HarshCasper** [comment]
LGTM!

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**